examples/with-webpack-bundle-size-analyzer/pages/contact.js

```diff
@@ -1 +0,0 @@
-export default () => <div>This is the contact page.</div>
```

Comment:
Why was this page removed? I think it's good to have a simple page like this to show its output in the bundle analyzer

Reply:
I removed the page because I didn't see it used anywhere in the application, but I wasn't thinking of it in the context of this example. It makes sense now that it is used to show its output in the bundle analyzer.